docs/operational_guide/kernel_tuning.md

```diff
@@ -0,0 +1,40 @@
+Kernel Tuning
+=============
+
+This document lists the Kernel tweaks `m3dbnode` needs to run well.
+
+## vm.max_map_count
+`m3dbnode` uses a lot of mmap-ed files for performance, as a result, you might need to bump `vm.max_map_count`. We suggest setting this value to `262144`, so you don’t have to come back and debug issues later.
+
+On Linux, you can increase the limits by running the following command as root:
+```
+sysctl -w vm.max_map_count=262144
+```
+
+To set this value permanently, update the `vm.max_map_count` setting in `/etc/sysctl.conf`.
+
+## vm.swappiness
+`vm.swappiness` controls how much the virtual memory subsystem will try to swap to disk. By default, the kernel configures this value to `60`, and will try to swap out items in memory even when there is plenty of RAM available to the system.
+
+For performance, we recommend sizing clusters such that `m3dbnode` is running on a substrate such that no-swapping is necessary. And therefore recommend setting the value of `vm.swappiness` to `1`. This tells the kernel to swap as little as possible, without altogether disabling swapping.
+
+On Linux, you can configure this by running the following as root:
+```
+sysctl -w vm.swappiness=1
+```
+
+To set this value permanently, update the `vm.swappiness` setting in `/etc/sysctl.conf`.
+
+
+## rlimits
+`m3dbnode` also can use a high number of files and we suggest setting a high max open number of files due to per partition fileset volumes.
```

Comment:
The explanations of why seem a little out  of place. I understand the desire to include them but it really breaks the flow.  If you want to keep them in could we add like a `### Rationale` subheading to each of these? That way you can just scan through and it just lists variables and what our recommended value is and if you care you can read the rationale

Reply:
the `vm.max_map_count` section does the same thing - explain why we need to tweak a value and then what the value should be tweaked to.

Reply:
done